classes/Leth/IPAddress/IPv6/Address.php

```diff
@@ -237,6 +232,12 @@ public function format($mode)
 				}
 				break;
 
+			case IPv6\Address::FORMAT_MAPPED_IPV4:
```

Comment:
It might be nice to have a format mode which will do mapped-ipv4 if possible, else fall back to abbreviated.